mappings/net/minecraft/util/math/Matrix4fStack.mapping

```diff
@@ -0,0 +1,8 @@
+CLASS net/minecraft/class_4587 net/minecraft/util/math/Matrix4fStack
```

Comment:
given matrixes serves as transformations, should we call it transformation stack etc (name from actual usage?)

Reply:
Maybe, but I think `MatrixStack` is more familiar because it matches the old GL `pushMatrix` and `popMatrix`. Based on a vanilla string, Mojang seems to be calling it "pose stack".